examples/atari/src/atari_wrappers.py

```diff
@@ -304,6 +305,72 @@ def __getitem__(self, i):
         return self._force()[..., i]
 
 
+class RawTrajectoryWrapper(Wrapper):
+    def __init__(
+            self,
+            env,
+            allow_early_resets=False,
+            reset_keywords=(),
+            info_keywords=()
+    ):
+        super().__init__(env)
+        self.reset_keywords = reset_keywords
+        self.info_keywords = info_keywords
+        self.allow_early_resets = allow_early_resets
+        self.needs_reset = True
+        self.observations = []
+        self.actions = []
+        self.rewards = []
+        self.dones = []
+
+    def reset(self, **kwargs):
+        self.reset_state()
+        for k in self.reset_keywords:
+            v = kwargs.get(k)
+            if v is None:
+                raise ValueError(
+                    "Expected you to pass kwarg %s into reset" % k
+                )
+            self.current_reset_info[k] = v
+        initial_state = self.env.reset(**kwargs)
+        self.observations = [initial_state, ]
```

Comment:
what do you think about optimizing the code with DynamicArray?
just like it was done on [`trajectory_sampler`](https://github.com/catalyst-team/catalyst/blob/master/catalyst/rl/core/trajectory_sampler.py#L37)
this should unite buffers logic and make it easy to support it in future